fc-exploitation/src/service-provider/service-provider.controller.ts

```diff
@@ -131,29 +130,33 @@ export class ServiceProviderController {
   @UseInterceptors(new FormErrorsInterceptor(`/service-provider/create`))
   @UsePipes(new ValidationPipe({ transform: true, whitelist: true }))
   async createServiceProvider(
-    @Body() createServiceProviderDto: ServiceProviderDto,
+    @Body() createServiceProviderRawDto,
```

Comment:
suggestion: I am unsure that you should remove validation & transformation of the body